build-tools/xaprepare/xaprepare/Application/GeneratedMonodroidCmakeFiles.cs

```diff
@@ -271,7 +271,7 @@ void WriteShellRuntimeCommand (StreamWriter sw, IEnumerable<string> abis, CmakeB
 				sw.WriteLine ($"{indent}__FORCE=\"${{force}}\"");
 				if (!command.IsHost) {
 					sw.WriteLine ($"{indent}__NATIVE_ABI={abi}");
-					sw.WriteLine ($"{indent}__NATIVE_API_LEVEL=${{{minApiLevel}}}");
```

Comment:
…how did this ever work?  Wouldn't it result in e.g. `__NATIVE_API_LEVEL=${21}`, which is non-sensical?

Reply:
Yeah, made me wonder too